Classes/game.rb

```diff
@@ -0,0 +1,16 @@
+require 'active_support/all'
+require './Classes/item'
+
+class Game < Item
+  attr_reader :multiplayer, :last_played_at
+
+  def initialize(publish_date, multiplayer, last_played_at)
+    super(publish_date)
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+  end
+
+  def can_be_archived?
+    super && last_played_at < 2.years.ago
```

Comment:
I think the correct syntax to find the date is not applied. It would be better if you set a correct date and then write code to find if 'last_played_at' is more than 2 years ago or not.
```suggestion
    now = Date.now
    time_elapsed = (now - last_played_at).to_i
```

Reply:
I have checked this `last_played_at < 2.years.ago`. If last_played_at >= '2021/03/22', it will return false.

Reply:
Alright 👍